osu.Game.Rulesets.Tau/Objects/Drawables/DrawableBeat.cs

```diff
@@ -20,6 +20,7 @@ public class DrawableBeat : DrawableTauHitObject, IKeyBindingHandler<TauAction>
     {
         public CompositeDrawable Box;
         public Container IntersectArea;
+        public float HitDistance = -.1f;
```

Comment:
This could be a constant, with a flag being used to get the inversed value. Totally up to you tho. 

```cs
HitDistance * (inversed ? -1 : 1)
```